### ISSUE TITLE
-[CalStoreIcon isFetchingImage]: unrecognized selector sent to instance

### ISSUE THREAD

**Issue by mpashnyov** [schedjoules/iOS-SDK#1]
Crash when checking image Fetching state
Revision: 0e98d45afde88308f37f3fcfa86258d6888771df

``` objc
request = [CalStorePageRequest requestWithStartPage];
[request fetchPageWithCompletionHandler:^(CalStorePage *page, NSError *error) {
        CalStorePageSection *section = page.sections.firstObject;
        CalStorePageItem *item = section.items.firstObject;
        @try {
            //-[CalStoreIcon isFetchingImage]: unrecognized selector sent to instance
            __unused BOOL imageFetching2 = [item.icon isFetchingImage];
        } @catch (NSException *exception) {
        }
}];
```

[TestCrash.zip](https://github.com/schedjoules/iOS-SDK/files/272938/TestCrash.zip)

**Comment by rutgerg** [comment]
Txs for reporting this. The cause has been identified and solved. We'll release an updated SDK tomorrow morning.

**Comment by rutgerg** [comment]
Solved in v2.0.1

**Comment by rutgerg** [comment]
Issue moved to [schedjoules/backlog #156](https://github.com/schedjoules/backlog/issues/156) via [**ZenHub**](https://www.zenhub.com/)
